src/controllers/autoCompletion.controller.js

```diff
@@ -73,3 +90,22 @@ exports.getAutoCompletions = async function (req, res, next) {
     });
   }
 };
+
+exports.deleteAutoCompletions = async function (req, res, next) {
+  const { historyToDelete } = req.query;
+  const { accessToken } = req.cookies;
+  const decodedToken = jwt.decode(accessToken);
+  const { userId } = decodedToken;
+
+  const user = await User.findOne({
+    _id: userId,
+  });
+  const privateHistoryIndex = user.searchHistory.indexOf(historyToDelete);
+
+  user.searchHistory.splice(privateHistoryIndex, 1);
+  user.save();
```

Comment:
await 붙여줘야 할 것 같습니다!
```suggestion
  await user.save();
```

Reply:
꼼꼼하지 못했네요. 수정하겠습니다!